examples/cuda_aware_mpi/multiple_devices.py

```diff
@@ -14,6 +14,9 @@
 size = comm.Get_size()
 if size != 2:
     raise RuntimeError("run this script with 2 processes: mpiexec -n 2 ...")
+device_count = cupy.cuda.runtime.getDeviceCount()
+if size > device_count:
+    raise RuntimeError("need {} GPUs, found {}".format(size, device_count))
```

Comment:
Error message is not clear.
Lets say something like
"This example only works with a number of processes equal to the number of GPUs in the local rank"

Reply:
Can we just say "this script requires 2 GPUs"?

Reply:
ok